src/helpers/scrollAnimationProvider.ts

```diff
@@ -6,23 +6,25 @@ import classnames, {
 import useBreakpoints from 'hooks/useBreakpoints'
 import useScrollPercent from 'hooks/useScrollPercent'
 
-export default function (animationName: string) {
+export default function (animationName: string, blockVisible = true) {
```

Comment:
We remove the `style={}` if the block is not visible, this saves a lot of memory and gpu